### PR TITLE
Use controller util to get all workloads in the cluster

### PR DESCRIPTION
**Pull request by makoscafee**
## Checklist

* [ ] I have updated the CHANGELOG of any plugins updated
* [ ] I have updated the version file of any plugins updated
* [ ] I have added tests wherever appropriate
* [ ] I have labeled this PR with a label per plugin updated

### CODE REVIEW

**Review by vitorvezani — APPROVED** [review]
LGTM! Very generic, clean and concise 🔥

**Review by ivanfetch-fw — APPROVED** [review]
Wow - that's quite an improvement that replaces a lot of internal plugin code to track / map / determine individual "top controllers"  by our `GetAllTopControllers` function!

File: plugins/workloads/pkg/info.go [hunk 1]
```diff
@@ -123,250 +127,32 @@ func CreateResourceProviderFromAPI(ctx context.Context, kube kubernetes.Interfac
...
-			ownerController := controllerMap[ownerUID]
-			if ownerController != nil {
-				if item.Status.StartTime != nil && item.Status.CompletionTime != nil {
-					cronChildren[ownerUID] = append(cronChildren[ownerUID], jobMetadata{
-						startTime: item.Status.StartTime.Time,
-						endTime:   item.Status.CompletionTime.Time,
-					})
-				}
-				continue
+			for _, ctn := range pd.Spec.Containers {
```
Comment (rbren) [inline plugins/workloads/pkg/info.go:148]: I think we'll end up getting the same containers for each pod - we really just want to know what the container names are. 

You could use [`GetPodSpec`](https://github.com/FairwindsOps/controller-utils/blob/e9a28b6e43fcb89e2d304d5004566c8c5c383181/pkg/controller/pod.go#L20) to get the container names.

**Review by rbren — CHANGES_REQUESTED** [review]
Thanks Barnie! Love to see all that deleted code.

I think we need to change the container logic a bit though

**Review by rbren — APPROVED** [review]

**Pull request merged**